pkg/cache/internal/informers_map.go

```diff
@@ -158,24 +164,42 @@ func (ip *InformersMap) Start(ctx context.Context) error {
 		defer ip.mu.Unlock()
 
 		// Set the stop channel so it can be passed to informers that are added later
-		ip.stop = ctx.Done()
+		ip.ctx = ctx
```

Comment:
Should we create a sub-context here with a cancel function?

Reply:
When would we cancel this sub-context?